docs/security.html

```diff
@@ -273,6 +275,24 @@
     sasl.kerberos.service.name="kafka"</pre>
             </li>
         </ol></li>
+
+    <li><h4><a id="security_rolling_upgrade" href="#security_rolling_upgrade">Incorporating Security Features in a Running Cluster</a></h4>
+        You can upgrade a running cluster to be secured by one of the supported protocols discussed previously. This is done in two phases:
+        <ul>
+            <li>Rolling upgrade the cluster to open an additional, secured port.</li>
+            <li>Rolling upgrade again to close the plaintext port.</li>
```

Comment:
Again, not sure about the usage of `upgrade` in the two lines above. "Rolling bounce" maybe?